src/data.yaml

```
@@ -1389,6 +1389,19 @@ jobs:
         long: 13.4877535
     review:
     rating:
+  - name: eaternity
+    website: https://eaternity.org
+    jobs: https://eaternity.org/about/#job-offer
+    description: CO2 calculation for food.
+    remote: false
+    speculative: false
+    field: Carbon Analysis of Foods
```

There is just this general category

```suggestion
    field: Carbon Analysis, Capture and Offsetting
```